ozaria/site/components/interactive/PageInteractive/common/BaseInteractiveLayout.vue

```diff
@@ -1,9 +1,9 @@
 <script>
-  import BaseInteractiveTitle from './BaseInteractiveTitle'
+  import InteractiveTitle from './InteractiveTitle'
```

Comment:
Why did we change this to InteractiveTitle?

Reply:
It binds to the Vuex store so it's no longer a presentational component.

Reply:
Sorry, could not understand..

Reply:
I think I understand, is it because we renamed BaseInteractiveTitle to InteractiveTitle? It looks like so when looking at the branch locally, but in the PR the file still seems to be named BaseInteractiveTitle.. am I missing something?

Reply:
Yup thats right we renamed the file.  I just checked locally and in the branch on GitHub and am seeing the renamed file.  Could have been a PR versioning issue?